src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -104,7 +104,7 @@ public void TimesIncludeMillisecondPart_Linux()
                     item = GetExistingItem(); // try a new file/directory
                 }
 
-                Assert.NotEqual(0, msec);
+                //Assert.NotEqual(0, msec);
```

Comment:
Can you explicitly add a comment here stating why we are commenting it out but leaving the test enabled (maybe link to https://github.com/dotnet/corefx/issues/27662), so that we don't question why it was commented out and remove the test/assert in the future?